packages/orchestration/src/util/request-config.test.ts

```diff
@@ -0,0 +1,155 @@
+import { createLogger } from '@sap-cloud-sdk/util';
+import { jest } from '@jest/globals';
+import {
+  addStreamOptions,
+  addStreamOptionsToLlmModuleConfig,
+  addStreamOptionsToOutputFilteringConfig
+} from './request-config.js';
+import { buildAzureContentSafetyFilter } from './filtering.js';
+import type {
+  ModuleConfigs,
+  OrchestrationConfig
+} from '../client/api/schema/index.js';
+import type {
+  OrchestrationModuleConfig,
+  StreamOptions
+} from '../orchestration-types.js';
+describe('stream util tests', () => {
```

Comment:
Honestly I imagine this is just a refactor, but going forward I would not mix things together.